site/source/docs/optimizing/Optimizing-Code.rst

```diff
@@ -44,9 +44,13 @@ Compiling source files to object files works as you'd expect in a native build s
 Link Times
 ==========
 
-To skip extra optimization work at link time, link with ``-O0`` or ``-O1``. It
-is ok to link with those flags even if the source files were compiled with a
-different optimization level.
+To skip extra optimization work at link time, link with ``-O0`` or ``-O1``. In
+those modes Emscripten focuses on faster iteration times.
```

Comment:
It might be worth extending this sentence right up front with a mention of `WASM_BIGINT`. Perhaps the first sentence could be something like:

"To get the fastest link times use -O0 or -O1 and -s WASM_BIGINT. In these optimization modes, emscripten will trade output code quality for faster link times. Using WASM_BIGINT will allow emscripten to skip post-processing of the wasm (see below for details)"

Reply:
+1 to what Derek said, `-sWASM_BIGINT` was also the first thing that came to my mind when I saw this.

Reply:
`WASM_BIGINT` is already mentioned a little lower down in this section. Is that not good enough there?

Reply:
IMO WASM_BIGINT is basically the tl;dr of the whole section. If you only read one thing about making link times faster it would be that, because currently it's the most impactful thing you can do.

Reply:
Ok, I moved it up and rewrote the whole section. There is now a paragraph on opt settings, on non-opt settings, and on the flag to verify link speed. How does it look?

Reply:
ping @dschuff @bmeurer